apps/condo/migrations/20230221010326-0223_incident_incidentclassifier_and_more.js

```diff
@@ -14,7 +14,7 @@ exports.up = async (knex) => {
 --
 -- Create model incident
 --
-CREATE TABLE "Incident" ("number" integer NOT NULL UNIQUE, "details" text NOT NULL, "status" varchar(50) NOT NULL, "textForResident" text NULL, "workStart" timestamp with time zone NOT NULL, "workFinish" timestamp with time zone NULL, "isScheduled" boolean NOT NULL, "isEmergency" boolean NOT NULL, "hasAllProperties" boolean NOT NULL, "id" uuid NOT NULL PRIMARY KEY, "v" integer NOT NULL, "createdAt" timestamp with time zone NULL, "updatedAt" timestamp with time zone NULL, "deletedAt" timestamp with time zone NULL, "newId" uuid NULL, "dv" integer NOT NULL, "sender" jsonb NOT NULL, "createdBy" uuid NULL, "organization" uuid NOT NULL, "updatedBy" uuid NULL);
+CREATE TABLE "Incident" ("number" integer NOT NULL UNIQUE, "details" text NOT NULL, "status" varchar(50) NOT NULL, "textForResident" text NULL, "workStart" timestamp with time zone NOT NULL, "workFinish" timestamp with time zone NULL, "workType" varchar(50) NULL, "hasAllProperties" boolean NOT NULL, "id" uuid NOT NULL PRIMARY KEY, "v" integer NOT NULL, "createdAt" timestamp with time zone NULL, "updatedAt" timestamp with time zone NULL, "deletedAt" timestamp with time zone NULL, "newId" uuid NULL, "dv" integer NOT NULL, "sender" jsonb NOT NULL, "createdBy" uuid NULL, "organization" uuid NOT NULL, "updatedBy" uuid NULL);
```

Comment:
you have lots of potentially searchable fields here like `status`, `number`, `workStart`, `workFinish`, etc. but no indexes added, that will make such searching very slow.

Reply:
If it's really needed, we'll add it later